Source/Core/VideoCommon/OnScreenDisplay.h

```diff
@@ -48,7 +56,7 @@ constexpr u32 VERY_LONG = 10000;
 // On-screen message display (colored yellow by default)
 void AddMessage(const std::string& message, u32 ms = Duration::SHORT, u32 rgba = Color::YELLOW);
 void AddTypedMessage(MessageType type, const std::string& message, u32 ms = Duration::SHORT,
-                     u32 rgba = Color::YELLOW);
+                     u32 rgba = Color::CYAN);
```

Comment:
I don't like this change. IMO defaults between `AddMessage` and `AddTypedMessage` should be the same. Override with `MessageType` if needed.